watchtower/blob/justice_kit_packet.go

```diff
@@ -35,6 +36,17 @@ const (
 	//    commit to-remote sig:           64 bytes, maybe blank
 	V0PlaintextSize = 274
 
+	// V1PlaintextSize is the plaintext size of a version 1 encoded blob.
+	//    sweep address length:            1 byte
+	//    padded sweep address:           42 bytes
+	//    revocation pubkey:              32 bytes
+	//    local delay pubkey:             32 bytes
+	//    commit to-local revocation sig: 64 bytes
+	//    hash of to-local delay script:  32 bytes
+	//    commit to-remote pubkey:        33 bytes, maybe blank
+	//    commit to-remote sig:           64 bytes, maybe blank
+	V1PlaintextSize = 300
```

Comment:
Any reason to _not_ just make this version TLV based? When we made the original format, I don't think stuff like `BigSize` had been fully conceptualized yet.